src/ProjectOrigin.VerifiableEventStore.Tests/BatchProcessorJobTests.cs

```diff
@@ -17,8 +17,8 @@ public async Task Will_Finalize_Full_Batch()
         var blockId = new Fixture().Create<string>();
         var transactionId = new Fixture().Create<string>();
         var blockChainConnector = new Mock<IBlockchainConnector>();
-        var batcherOptions = new BatcherOptions() { BatchSizeExponent = 2 };
-        var eventStore = new MemoryEventStore(batcherOptions);
+        var batcherOptions = new VerifiableEventStoreOptions() { BatchSizeExponent = 2 };
```

Comment:
Is the BatchSizeExponent hardcoded here ?!? or is the = the default value

Reply:
it is a test case, but I just saw we have negative tests, I will add some more.

Reply:
changed it to a  theory, with multiple test sweeps